src/components/engagement/events/internship-engagement-updated.event.ts

```diff
@@ -0,0 +1,9 @@
+import { ISession } from '../../../common';
+import { InternshipEngagement } from '../dto';
+
+export class InternshipEngagementUpdatedEvent {
+  constructor(
+    readonly engagement: InternshipEngagement,
```

Comment:
For the updated events I've been giving the `updated` & `previous` object as well as `updates` (changes).
https://github.com/SeedCompany/cord-api-v3/blob/cbb0a2dc2aa6a603a3dbc3e24ad220f1c39ac460/src/components/project/events/project-updated.event.ts#L6-L8
Let's do that in both of these updates too.